tests/integrations/kernels/scattermoe_lora/test_ep_sentinel_skip.py

```
@@ -0,0 +1,115 @@
+from axolotl.integrations.kernels.libs.scattermoe_lora.experts import (  # noqa: E402
+    scattermoe_experts_forward,
+    scattermoe_experts_forward_ep,
+)
+from axolotl.integrations.kernels.libs.scattermoe_lora.layers import (  # noqa: E402
+    peft_lora_to_scattermoe,
+)
+
+DEV = "cuda"
+
+
+def _module(E, H, I, dt, rank, monkeypatch):
```

Ruff (0.15.15)

[error] Ambiguous variable name: `I`

(E741)